### PR TITLE
Fix(Consents): Fix email reply to.

### PR DESCRIPTION
**Pull request by lauryndbrown**
Fixes https://github.com/carpentries/amy/issues/2030

### CODE REVIEW

File: amy/autoemails/actions.py [hunk 1]
```diff
@@ -1153,15 +1153,15 @@ def recipients(self) -> Optional[str]:
         """Assuming self.context is ready, overwrite email's recipients
```
Comment (lauryndbrown) [inline amy/autoemails/actions.py:1153]: TODO `recipients(self)` does not return a string change.

Comment (lauryndbrown) [reply amy/autoemails/actions.py:1153]: Ticketed here: https://github.com/carpentries/amy/issues/2037

File: amy/autoemails/actions.py [hunk 2]
```diff
@@ -105,9 +105,9 @@ def bcc_recipients(self) -> Optional[str]:
         Action."""
         return None
 
-    def reply_to(self) -> str:
+    def reply_to(self) -> Optional[Tuple[str]]:
```
[the remark this answers is not in the document]
Comment (pbanaszkiewicz) [reply amy/autoemails/actions.py:108]: This is okay - see my comments in another thread below.

File: amy/autoemails/fixtures/templates_triggers.json [hunk 3]
```diff
@@ -190,7 +190,7 @@
             "from_header": "amy-no-reply@carpentries.org",
             "cc_header": "",
             "bcc_header": "",
-            "reply_to_header": "{{ regional_coordinator_email.0 }}",
+            "reply_to_header": "",
```
Comment (pbanaszkiewicz) [inline amy/autoemails/fixtures/templates_triggers.json:193]: Note to myself: this change will require updating template manually in production.

**Review by pbanaszkiewicz — APPROVED** [review]
Looking good, maybe small detail to change in one try-except. Good job with all these type hints @lauryndbrown!

File: amy/autoemails/actions.py [hunk 4]
```diff
@@ -1153,15 +1153,15 @@ def recipients(self) -> Optional[str]:
...
             return None
 
     def all_recipients(self) -> str:
         """If available, return string of all recipients."""
         try:
-            person_email = self.context_objects["person_email"]
-            return person_email
+            return ", ".join(self.recipients())
         except (KeyError, AttributeError):
             return ""
```
[the remark this answers is not in the document]
Comment (lauryndbrown) [reply amy/autoemails/actions.py:1166]: You're right good catch thank you.

**Comment by pbanaszkiewicz** [comment]
@lauryndbrown I think you need to rebase against `develop`. I remember I fixed this error: https://github.com/carpentries/amy/runs/3330839656?check_suite_focus=true

**Pull request merged**